[ bug ] Doesn't build on FreeBSD

**tgpt version**

commit 867d35d

**Bug description**

Doesn't build on FreeBSD:

```console
❯ go build -v
github.com/aandrew-me/tgpt/v2/src/structs
golang.design/x/clipboard
# golang.design/x/clipboard
/home/yonas/go/pkg/mod/golang.design/x/clipboard@v0.7.1/clipboard.go:108:15: undefined: initialize
/home/yonas/go/pkg/mod/golang.design/x/clipboard@v0.7.1/clipboard.go:119:14: undefined: read
/home/yonas/go/pkg/mod/golang.design/x/clipboard@v0.7.1/clipboard.go:139:18: undefined: write
/home/yonas/go/pkg/mod/golang.design/x/clipboard@v0.7.1/clipboard.go:154:9: undefined: watch
github.com/aandrew-me/tgpt/v2/src/client
github.com/aandrew-me/tgpt/v2/src/utils
github.com/aandrew-me/tgpt/v2/src/providers/gemini
github.com/aandrew-me/tgpt/v2/src/providers/groq
github.com/aandrew-me/tgpt/v2/src/providers/ollama
github.com/aandrew-me/tgpt/v2/src/providers/deepseek
github.com/aandrew-me/tgpt/v2/src/providers/isou
github.com/aandrew-me/tgpt/v2/src/providers/koboldai
github.com/aandrew-me/tgpt/v2/src/providers/openai
github.com/aandrew-me/tgpt/v2/src/providers/pollinations
github.com/aandrew-me/tgpt/v2/src/providers/sky
github.com/aandrew-me/tgpt/v2/src/imagegen/arta
github.com/aandrew-me/tgpt/v2/src/providers/kimi
github.com/aandrew-me/tgpt/v2/src/imagegen/pollinations
github.com/aandrew-me/tgpt/v2/src/imagegen
github.com/aandrew-me/tgpt/v2/src/providers
github.com/aandrew-me/tgpt/v2/src/search
```

--- a/src/clipboard/clipboard.go
+++ b/src/clipboard/clipboard.go
@@ -1,0 +1,1 @@
+package clipboard

--- a/src/clipboard/clipboard_freebsd.go
+++ b/src/clipboard/clipboard_freebsd.go
@@ -1,0 +1,9 @@
+//go:build freebsd
+// +build freebsd
+
+package clipboard
+
+// CopyToClipboard is a no-op on FreeBSD as clipboard functionality is not supported
+func CopyToClipboard(text string) {
+	// Clipboard functionality not available on FreeBSD
+}

--- a/src/clipboard/clipboard_other.go
+++ b/src/clipboard/clipboard_other.go
@@ -1,0 +1,20 @@
+//go:build !freebsd
+// +build !freebsd
+
+package clipboard
+
+import (
+	"fmt"
+
+	"golang.design/x/clipboard"
+)
+
+// CopyToClipboard copies text to the system clipboard
+func CopyToClipboard(text string) {
+	err := clipboard.Init()
+
+	if err == nil {
+		clipboard.Write(clipboard.FmtText, []byte(text))
+		fmt.Println("Copied command to clipboard")
+	}
+}

--- a/src/helper/helper.go
+++ b/src/helper/helper.go
@@ -13,6 +13,7 @@ import (
 	"time"
 
 	"github.com/aandrew-me/tgpt/v2/src/client"
+	"github.com/aandrew-me/tgpt/v2/src/clipboard"
 	"github.com/aandrew-me/tgpt/v2/src/imagegen/arta"
 	"github.com/aandrew-me/tgpt/v2/src/providers"
 	"github.com/aandrew-me/tgpt/v2/src/search"
@@ -20,7 +21,6 @@ import (
 	"github.com/aandrew-me/tgpt/v2/src/utils"
 	http "github.com/bogdanfinn/fhttp"
 	"github.com/fatih/color"
-	"golang.design/x/clipboard"
 
 	"github.com/olekukonko/ts"
 
@@ -817,13 +817,7 @@ func MakeRequestAndGetData(input string, params structs.Params, extraOptions str
 				if userInput == "y" || userInput == "" {
 					ExecuteCommand(ShellName, ShellOptions, fullText)
 				} else {
-					err := clipboard.Init()
-
-					if err == nil {
-						clipboard.Write(clipboard.FmtText, []byte(fullText))
-
-						fmt.Println("Copied command to clipboard")
-					}
+					clipboard.CopyToClipboard(fullText)
 				}
 			}
 		}

